File: app/src/main/java/com/by5388/sy95306v2/bean/shenyang/TrainDetail.java
package com.by5388.sy95306v2.bean.shenyang;

/**
 * 沈阳铁路车次查询的结果
 *
 * @author by5388  on 2018/6/6.
 */
@SuppressWarnings("all")
public class TrainDetail {

    /**
     * STNO : 01
     * ATIME : ----
     * SNAME : 深圳北
     * STIME : 1105
     * STCODE : D7414
     */

    private String STNO;
    private String ATIME;
    private String SNAME;
    private String STIME;
    private String STCODE;

    public String getSTNO() {
        return STNO;
    }

    public void setSTNO(String STNO) {
        this.STNO = STNO;
    }

    public String getATIME() {
        return ATIME;
    }

    public void setATIME(String ATIME) {
        this.ATIME = ATIME;
    }

    public String getSNAME() {
        return SNAME;
    }

    public void setSNAME(String SNAME) {
        this.SNAME = SNAME;
    }

    public String getSTIME() {
        return STIME;
    }

    public void setSTIME(String STIME) {
        this.STIME = STIME;
    }

    public String getSTCODE() {
        return STCODE;
    }

    public void setSTCODE(String STCODE) {
        this.STCODE = STCODE;
    }
}

File: sy95306v1kotlin/src/main/java/com/by5388/sy95306/kotlin/bean/TrainDetail.kt
package com.by5388.sy95306.kotlin.bean

import com.google.gson.annotations.SerializedName

/**
 * @author  by5388  on 2019/1/5.
 */
data class TrainDetail(@SerializedName("STNO") val number: String,
                       @SerializedName("ATIME") val arriveTime: String,
                       @SerializedName("SNAME") val stationName: String,
                       @SerializedName("STIME")  val leaveTime: String,
                       @SerializedName("STCODE")   val trainCode: String
)